.aws-infrastructure/environment-template.mustache.yml

```diff
@@ -15,6 +15,7 @@ Parameters:
       - dev
       - stg
       - prd
+      - gha
```

Comment:
We probably don't need to keep the `gha` options in here, but keeping them is fine.